public/assets-new-ui/styles.css

```diff
@@ -0,0 +1,260 @@
+body {
+  background: white;
+  margin: 0;
+  padding: 0;
+}
+
+/*= HEADER */
+
+header[role="banner"] {
+  background-color: #e9eff1;
+  border-bottom: 1px solid #dcdcdc;
+}
+
+.glogo {
+  width: 69px;
+  height: 69px;
+}
+
+/*= FOOTER */
+
+footer[role="contentinfo"] {
+  background: #333;
+  color: #999;
+}
+
+/* Support navigation */
+.support ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.support a {
+  color: white;
+  text-decoration: none;
+}
+
+/*= CONTENT */
+.content {
+  background: white;
+  padding: 0 10px;
+}
+
+.contribution-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.form__field {
+  align-items: stretch;
+  display: flex;
+  flex-direction: column;
+}
+
+/*= BREAKPOINTS */
+
+@media (min-width: 720px) {
```

Comment:
Which [breakpoint](https://github.com/guardian/support-frontend/blob/master/assets/stylesheets/gu-sass/breakpoints.scss) does this correspond to?

Reply:
Oops, that should be 740 indeed 👍